guide/menu.md

```diff
@@ -51,6 +51,10 @@
     - [MATLAB](matlab.md)
     - [Controller Plugin](controller-plugin.md)
     - [Webots Plugin](webots-plugin.md)
+- [Simulation Export](simulation-export.md)
```

Comment:
I believe we should add the word "Web" in the title of this chapter... Maybe "Web interface" or "Web Export", "Export Simulation to the Web"...